oak_runtime/src/node/http/server.rs

```diff
@@ -612,5 +532,5 @@ fn to_hyper_response(http_response: HttpResponse) -> Result<Response<Body>, Http
     builder
         .status(http_response.status as u16)
         .body(Body::from(http_response.body))
-        .map_err(|err| HttpError::HttpResponse(format!("Could not build response: {}", err)))
+        .map_err(|err| anyhow!("Could not build response: {}", err))
```

Comment:
Does `context` work for this and other cases in which we want to propagate the original error?

Reply:
Yes. Fixed.